task-3/tests/user.test.ts

```diff
@@ -0,0 +1,149 @@
+import dotenv from "dotenv";
+dotenv.config();
+
+import request from "supertest";
+import app from "../app";
+
+import { AuthService } from "../services/authentication";
+import { UserService } from "../services/user";
+
+const jwtToken = AuthService.generateAccessToken({
+  username: "newUser123",
+  password: "somePassword",
+});
+
+describe("Test User Path", () => {
+  test.each([
+    {
+      path: "/user/suggested?login=test",
+      expected: 400,
+    },
+    {
+      path: "/user/suggested?limit=3",
+      expected: 400,
+    },
+    {
+      path: "/user/suggested?limit=3&login=test",
+      expected: 200,
+    },
+  ])("Requesting suggested user", async ({ path, expected }) => {
+    const suggestedUsers: any[] = await UserService.getSuggestedUsers({
+      login: "test",
+      limit: 3,
+    });
+    const data = await request(app)
+      .get(path)
+      .set("Authorization", "Bearer " + jwtToken);
+
+    if (expected === 200) {
```

Comment:
Ok, I didn't notice all the status code checks at first :)
Consider using the same "http-status-code" you've added for business logic already - that was the primary goal actually :)